server/src/actors/healthcheck.rs

```diff
@@ -0,0 +1,67 @@
+use ::actix::prelude::*;
```

Comment:
Is the leading `::` required here?
Can be grouped with the next section.

Reply:
no clue, done